GitExtensions/Program.cs

```diff
@@ -67,7 +67,7 @@ private static void Main()
             {
                 DiagnosticsClient.Initialize(ThisAssembly.Git.IsDirty);
 
-                if (!Debugger.IsAttached)
+                ////if (!Debugger.IsAttached)
```

Comment:
Why just comment out?
Remove and add comment instead?

Reply:
I left it for whom has added this arguable behavior in order make them be able to easily restore it temporarily.

Reply:
Then describe in a comment it, no ////

Reply:
This is done so when a developer runs with a debugger attached (e.g. from a VS) and there is an exception the debugger breaks on the exception, and not terminates the process.
Please revert this change.

Reply:
It is opposite: If this condition is active, the application just terminates. One cannot debug the `BugReportInvoker` etc.

Reply:
You are absolute right. I got the git error wrong. I have replaced this with a check for `ExitedSuccessfully`.
Else stashed untracked files were not displayed anymore.
![grafik](https://user-images.githubusercontent.com/36601201/118717815-d06cee80-b826-11eb-8d7d-f374e7c37e82.png)

Reply:
> This is done so when a developer runs with a debugger attached (e.g. from a VS) and there is an exception the debugger breaks on the exception, and not terminates the process.

🤔 In order to achieve _this_, I tick all exceptions but `TaskCanceledException`:

![grafik](https://user-images.githubusercontent.com/36601201/118723903-3f9a1100-b82e-11eb-85ec-cbeead43a9fa.png)

The `BugReportInvoker` kicks in later when the exception has bubbled up.